utils.sh

```diff
@@ -30,8 +30,8 @@ function create_repo() {
 
 function set_policy() {
   local region=$1
-  RepositoryPermissionPolicy=$(printenv | grep RepositoryPermissionPolicy | sed 's/.*=//')
-  if [ -n "$RepositoryPermissionPolicy" ]
+  RepositoryPermissionPolicy=$(printenv RepositoryPermissionPolicy)
```

Comment:
O_o that doesn't really make sense to me. Why are these two lines of code different than each other?

```
RepositoryPermissionPolicy=$(printenv RepositoryPermissionPolicy)
```

```
RepositoryPermissionPolicy=${RepositoryPermissionPolicy}
```

If RepositoryPermissionPolicy is set in the env, then why would `set -u` be a problem?